src/Tribe/Template.php

```diff
@@ -0,0 +1,320 @@
+<?php
+// Don't load directly
+defined( 'WPINC' ) or die;
```

Comment:
As a note: this prevents any autoloading not in a WordPress context to happen. While I know this is common WordPress practice one has to keep that in mind.